packages/auth-providers/clerk/setup/src/setupHandler.ts

```diff
@@ -26,17 +26,14 @@ export const handler = async ({ force: forceArg }: Args) => {
       '```title=".env"',
       'CLERK_PUBLISHABLE_KEY="..."',
       'CLERK_SECRET_KEY="..."',
-      'CLERK_JWT_KEY="-----BEGIN PUBLIC KEY-----',
-      '...',
-      '-----END PUBLIC KEY-----"',
```

Comment:
@jtoar Just curious why you are removing this? Also right above it says:
> "You'll need to add three env vars to your .env file:". 

If you remove this should it be `two env vars` now?

Reply:
Thanks @o0charlie0o just fixed the text. I know at some point (here it looks like https://github.com/redwoodjs/redwood/pull/7528) I realized that this env var isn't necessary. Clerk doesn't make it easy to find in their dashboard (it's hidden under an "Advanced" toggle in API keys), and based on our [conversations with the Clerk team](https://github.com/redwoodjs/redwood/pull/4846#issuecomment-1076112316) and just by looking at the [code](https://github.com/clerkinc/javascript/blob/4e1bb2bd1f3cc933b1e8422849119e0aa16fdaa6/packages/backend/src/tokens/verify.ts#L43-L47), Clerk fetches it if it's not provided. But I'll confirm with the Clerk team that those network requests aren't rate limited.

You can still provide the key and the decoder will use it, it's just not necessary to get started, so that was the main motivation in removing it from the notes. Make sense/did that address your concerns?

Reply:
Yep! Thanks!